lib-test.js

```diff
@@ -1,23 +1,38 @@
-// const GoogleDriveStorage = require('./dist/models/GoogleDriveStorage.js');
-import { GoogleDriveStorage } from './dist/models/GoogleDriveStorage.js';
+import { StorageContext, StorageFactory } from './dist/index.js';
+
+const accessToken =
+	'ya29.a0AXooCguLHaKYRd1m3oc-bNvh9Q8kBbSFgfrM_ULtXt_l7wdMcyyRU31IgQAygNEOeimWlVMlbXLwyEyAowXAABUElGUiftTf-YhUiruW2VGg04k2X4-0n1ZU_hy7cukRIzUmjH5Smcbps2Yyb3p82PCJcimB5m_a3Tu8aCgYKAQ4SARMSFQHGX2Mi1330b5sRzCjyQrDomrNFYw0171';
+let fileId = '';
+const strategy = StorageFactory.getStorageStrategy('googleDrive', { accessToken });
+
+const storage = new StorageContext(strategy);
+
 async function createFolderAndUploadFile() {
-  try {
-    const storage = new GoogleDriveStorage('ADD_YOUR_ACCESS_TOKEN'); // please contact to @0marSalah to add your account to the project
-    const folderName = 'USER_UN(IQUE_KEY'; // need to discussed with team how we sill set the folder name
-    const folderId = await storage.createFolder(folderName);
+	try {
+		const folderName = 'USER_UNIQUE_KEY'; // need to discuss with team how we will set the folder name
```

Comment:
do we have a list of outstanding larger questions, that should be brought up with Dmitri?

Reply:
yeah actually this comment discussed in last meeting,